src/ui/Sort/Sort.ts

```diff
@@ -196,6 +196,108 @@ export class Sort extends Component {
     return _.any(this.options.sortCriteria, (sortCriteria: SortCriteria) => sortId == sortCriteria.toString());
   }
 
+  private findOrCreateRadioGroup() {
+    this.radioGroup = this.findRadioGroup();
+    if (!this.radioGroup) {
+      this.element.setAttribute('role', 'radiogroup');
+      this.radioGroup = this.element;
+    }
+  }
+
+  private createSortButton(innerText?: string) {
+    this.sortButton = $$('span').el;
+    this.sortButton.innerText = this.options.caption || innerText;
+    new AccessibleButton()
+      .withElement(this.sortButton)
+      .withEnterKeyboardAction(() => this.selectAndExecuteQuery())
+      .withArrowsAction((direction, e) => this.onArrowPressed(direction, e))
+      .withLabel(
+        this.canChangeDirection() ? this.getDirectionalLabel(this.initialDirection as VALID_DIRECTION) : this.getOmnidirectionalLabel()
```

Comment:
Why is the `as VALID_DIRECTION` needed?

Reply:
because the `initialDirection` getter returns `"" | VALID_DIRECTION`, but when `canChangeDirection()` returns `true`, it's guaranteed not to be `""`.